src/main/java/com/programmers/springweekly/controller/CustomerApiController.java

```diff
@@ -0,0 +1,68 @@
+package com.programmers.springweekly.controller;
+
+import com.programmers.springweekly.dto.customer.request.CustomerCreateRequest;
+import com.programmers.springweekly.dto.customer.response.CustomerListResponse;
+import com.programmers.springweekly.dto.customer.response.CustomerResponse;
+import com.programmers.springweekly.service.CustomerService;
+import com.programmers.springweekly.util.validator.CustomerValidator;
+import java.util.List;
+import java.util.NoSuchElementException;
+import java.util.UUID;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.validation.annotation.Validated;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/api/customer")
+@RequiredArgsConstructor
+public class CustomerApiController {
+
+    private final CustomerService customerService;
+
+    @PostMapping("/save")
+    public ResponseEntity<CustomerResponse> save(@Validated CustomerCreateRequest customerCreateRequest) {
+        CustomerValidator.validateCustomer(
+                customerCreateRequest.getCustomerName(),
+                customerCreateRequest.getCustomerEmail(),
+                customerCreateRequest.getCustomerType()
+        );
+
+        CustomerResponse customerResponse = customerService.save(customerCreateRequest);
+
+        return new ResponseEntity<>(customerResponse, HttpStatus.CREATED);
+    }
+
+    @GetMapping("/find")
+    public ResponseEntity<List<CustomerResponse>> getFindAll() {
+        CustomerListResponse customerListResponse = customerService.findAll();
+
+        return ResponseEntity.ok(customerListResponse.getCustomerList());
+    }
+
+    @GetMapping("/find/{id}")
+    public ResponseEntity<CustomerResponse> findById(@PathVariable("id") UUID customerId) {
+        CustomerResponse customerResponse = customerService.findById(customerId);
+
+        return ResponseEntity.ok(customerResponse);
+    }
+
+    @GetMapping("/delete/{id}")
```

Comment:
```suggestion
    @DeleteMapping("/{id}")
```

http method 종류에 대해 공부해봅시다!
https://inpa.tistory.com/entry/WEB-%F0%9F%8C%90-HTTP-%EB%A9%94%EC%84%9C%EB%93%9C-%EC%A2%85%EB%A5%98-%ED%86%B5%EC%8B%A0-%EA%B3%BC%EC%A0%95-%F0%9F%92%AF-%EC%B4%9D%EC%A0%95%EB%A6%AC